src/winetricks

```diff
@@ -14953,6 +14953,45 @@ load_openwatcom()
 
 #----------------------------------------------------------------
 
+w_metadata powershell apps \
+    title="Windows Powershell" \
+    publisher="Microsoft" \
+    year="2019" \
+    media="download" \
+    file1="PowerShell-6.1.6-win-x86.zip" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/WindowsPowerShell/v1.0/powershell.exe"
+
+load_powershell()
+{
+    if test "$W_ARCH" = "win64"; then
+        w_die "Only implemented for 32bit arch at the moment"
+    fi
+
+    w_warn "Installing Powershell into $W_SYSTEM32_DLLS_WIN/WindowsPowerShell/v1.0/"
+    w_warn "So that apps that require psh scripts to be executed would find it"
+    w_warn "Note that some functionality may not work without Windows Management Framework"
+
+    w_call dotnet462
+
+    w_download https://github.com/PowerShell/PowerShell/releases/download/v6.1.6/PowerShell-6.1.6-win-x86.zip 1dc690c9a1091d0ed5b71de903ab3cd7b58ba9638acdfeb9f9fe7031abaf47e9 PowerShell-6.1.6-win-x86.zip
+
+    w_override_dlls native powershell.exe
+
+    # remove builtin placeholders
+    rm -f "$W_SYSTEM32_DLLS"/powershell.exe
+
+
+    mkdir -p "$W_SYSTEM32_DLLS"/WindowsPowerShell/v1.0
+
+    w_try_7z  "$W_SYSTEM32_DLLS"/WindowsPowerShell/v1.0/ "${W_CACHE}/${W_PACKAGE}/${file1}"
+    w_try_cd "$W_SYSTEM32_DLLS"/WindowsPowerShell/v1.0
+    w_try cp -f pwsh.exe powershell.exe
+    for file in *\\*; do target="${file//\\//}"; mkdir -p "${target%/*}"; mv -v "$file" "$target"; done
```

Comment:
mkdir/mv should be behind w_try().

This also introduces a bashism, causing travis to fail.